index.html

```diff
@@ -492,6 +498,11 @@ <h1 class="page-header">Red October Management</h1>
 					data.Labels[i] = data.Labels[i].trim();
 					if (data.Labels[i] == "") { data.Labels.splice(i, 1); }
 				}
+				data.Usages = data.Usages.split(',');
+				for(var i=0, l=data.Usages.length; i<l; i++){
+					data.Usages[i] = data.Usages[i].trim();
+					if (data.Usages[i] == "") { data.Usages.splice(i, 1); }
```

Comment:
I've run into this same issue before. You can address it by changing it to:

    for(var i=0; i < data.Usages.length; i++) {
        data.Usages[i] = data.Usages[i].trim();
        if (data.Usages[i] === "") {
            data.Usages.splice(i, 1);
            i--;
        }
    }

I agree that it should be a separate PR. I'll create an issue for it and work on that